code/playerman/playercontrol.cpp

```diff
@@ -397,35 +397,37 @@ void do_view_slew()
 	view_modify(&chase_slew_angles, &Viewer_slew_angles_delta, PI_2, PI2/3);
 
 	// Check Track target
-	if (Viewer_mode & VM_TRACK) {
+	if (Viewer_mode & VM_TRACK && !Slew_locked) {
```

Comment:
The fact that slew-locked prevents tracking should also be documented somewhere.

Reply:
`VM_TRACK` is a variant of padlock mode, so I figured it would be implicit.  But I can make it implicit.

Reply:
That may not be evident to some modders, so it might be good to be explicit here